src/game/Inventory.cpp

```diff
@@ -1088,15 +1088,14 @@ void CheckForInventoryReplaceMe(Entity * io, Entity * old) {
  */
 bool TakeFromInventory(const Vec2s & pos) {
 	Entity * io = GetFromInventory(pos);
-	
+
 	if(io == NULL)
 		return false;
-	
-	if(g_secondaryInventoryHud.dragEntity(io, pos))
-		return true;
-	
-	g_playerInventoryHud.dragEntity(io, pos);
 
+	if(!g_secondaryInventoryHud.dragEntity(io, pos))
+		return false;
```

Comment:
We recently decided to use braces (`{}`) even for single line commits - see the [code style examples](http://wiki.arx-libertatis.org/Code_Style_Guide#Example). I know this didn't have any before, but if you are changing this line, might as well add them. Not that important though.